ivy/functional/frontends/numpy/sorting_searching_counting/sorting.py

```diff
@@ -3,11 +3,21 @@
 
 
 def argsort(
-    x,
-    /,
-    *,
-    axis=-1,
-    kind=None,
-    order=None,
+        x,
+        /,
+        *,
+        axis=-1,
+        kind=None,
+        order=None,
 ):
     return ivy.argsort(x, axis=axis)
+
+
+def argmax(x,
```

Comment:
These should match the names and order of parameters of the matching NumPy method, as described [here](https://numpy.org/doc/stable/reference/generated/numpy.argmax.html). Specifically, the `x` parameter should be named `a`.